### DISCUSSION TITLE
The role of "-Denable-pywrapper"

### DISCUSSION THREAD

**Discussion by Zcaic** [su2code/SU2#1957] [General]
Hello:
      What's  role of the compile options "-Denable-pywrapper" ? Does the python scripts like "parallel_computation.py" need "-Denable-pywrapper" to be on?

**Comment by bigfooted** [comment]
Hi,
No, these python scripts are independent from the python wrapper. The python scripts just call the SU2 executables, just open them and see what's going on. With the python wrapper you can interact between python and SU2 and create for instance user defined boundary conditions and initial conditions (under heavy development right now so many features are only available in the develop branch). For some examples on how to use the python wrapper, see:
https://github.com/su2code/SU2/tree/master/TestCases/py_wrapper

**Reply by Zcaic** [reply]
Thanks a lot！

**Reply by carlos-pereyra** [reply]
@bigfooted Is there a list of hardware that the pywrapper tests successfully pass?

The tests yield these errors.

1. zsh: segmentation fault  python run.py
2. ModuleNotFoundError: No module named 'pysu2ad'

Ty in advance! :)

I'm on:
* OS: Mac OSX M2
* C++ compiler and version: Apple clang version 14.0.3, Target: arm64-apple-darwin23.4.0
* Python Version: 3.9.19
* SWIG Version: 4.2.1
* SU2 Release 8.0.1 "Harrier"

         Option          Value
         ---------------------
         TecIO:          true
         CGNS:           true
         AD (reverse):   false
         AD (forward):   false
         Python Wrapper: true
         Intel-MKL:      false
         OpenBlas:       false
         PaStiX:         false
         Mixed Float:    false
         libROM:         false
         CoolProp:       false
         MLPCpp:         false
